packages/@react-spectrum/tree/package.json

```diff
@@ -37,16 +37,18 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
-    "@react-aria/focus": "^3.1.0",
-    "@react-aria/selection": "^3.1.0",
-    "@react-aria/virtualizer": "^3.1.0",
+    "@react-aria/button": "^3.9.3",
+    "@react-aria/i18n": "^3.10.2",
+    "@react-aria/tree": "3.0.0-alpha.1",
+    "@react-aria/utils": "^3.23.2",
+    "@react-spectrum/checkbox": "^3.9.4",
+    "@react-spectrum/style-macro-s1": "^3.5.3",
```

Comment:
oh hm, well it is the version that is set on the current style macro package but I can see your point. I'll to drop it down to `1.0.0-alpha.1` I guess since that package is private